rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1837,6 +1838,10 @@ public Integer voteFeePerKbChange(Transaction tx, Coin feePerKb) {
             return FEE_PER_KB_GENERIC_ERROR_CODE;
         }
 
+        if(!feePerKb.isPositive()){
```

Comment:
If isPositive does what it seems to do, then this line is preventing setting zero fees, which AFAIK it could be useful.